uml/backend-software-system/screen-handler.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+
+package "Screen Handler" {
+  class EventType {
+    type : String
+    sprite : Sprite
+  }
+
+  class SpritesReader {
+    +readSprites() : List<Sprite>
+  }
+
+  class ScreenManager {
+    -sprites : List<Sprite>
+    -mapper : ScreenMapper
+    -events : ScreenEvents
+    -settings : ScreenSettings
+    +showSprites(sprites : List<Sprite>)
+  }
+
+  class ScreenMapper {
+    +showSprites(sprites : List<Sprite>)
+    +updateSprite(sprite : Sprite)
+  }
+
+  class ScreenEvents {
+    +listenEvents() : void
+    +onEvent(event : EventType) : void
+  }
```

Comment:
This logic will be handled by "Input device monitor", so please erase this.

Reply:
Yeah, I had reviewed "Input Device Monitor" package and it's not necessary this one here.